### PR TITLE
Prebuilt rule customization — JSON diff [Classic]

### PR DESCRIPTION
**Pull request by joepeeples**
Contributes to https://github.com/elastic/security-docs/issues/4371 by updating the classic docs with info on the JSON diff for prebuilt rule updates.

Preview: [Install and manage Elastic prebuilt rules | Update Elastic prebuilt rules](https://security-docs_bk_4463.docs-preview.app.elstc.co/guide/en/security/master/prebuilt-rules-management.html#update-prebuilt-rules)

### Twin PR
* https://github.com/elastic/staging-serverless-security-docs/pull/245

### CODE REVIEW

**Review by nikitaindik — DISMISSED** [review]
Checked the changes and the preview. LGTM. Thanks @joepeeples!

File: docs/detections/prebuilt-rules-management.asciidoc [hunk 1]
```diff
@@ -96,12 +96,19 @@ Elastic regularly updates prebuilt rules to optimize their performance and ensur
...
 +
-TIP: To examine the details of a rule's latest version before you update it, select the rule name. This opens the rule details flyout.
-+
 [role="screenshot"]
 image::images/prebuilt-rules-update.png[The Rule Updates tab on the Rules page]
 
-. Do one of the following:
+. (Optional) To examine the details of a rule's latest version before you update it, select the rule name. This opens the rule details flyout.
++
+The *Updates* tab displays a side-by-side JSON comparison of the rule's *Base version* (what you currently have installed) and the *Update* version that you can choose to install. Deleted characters are in red; added characters are in green.
```
Comment (nikitaindik) [inline docs/detections/prebuilt-rules-management.asciidoc:104]: Suggestion: "deleted characters are highlighted red; added characters are highlighted green"

Comment (joepeeples) [reply docs/detections/prebuilt-rules-management.asciidoc:104]:
```suggestion
The *Updates* tab displays a side-by-side JSON comparison of the rule's *Base version* (what you currently have installed) and the *Update* version that you can choose to install. Deleted characters are highlighted in red; added characters are highlighted in green.
```

**Review by nikitaindik — APPROVED** [review]

**Comment by joepeeples** [comment]
Docs are approved, no further review or updates needed. OK to merge for classic docs.

**Pull request merged**